docs/api.md

```diff
@@ -0,0 +1,118 @@
+
+
+
+####  doAdministratorLogin() 
+Function to Do Admin Login In Joomla!
+
+####  doFrontEndLogin() 
+Function to Do frontend Login in Joomla!
+
+####  installJoomla() 
+Installs Joomla
+
+####  installJoomlaRemovingInstallationFolder() 
+Install Joomla removing the Installation folder at the end of the execution
+
+####  installJoomlaMultilingualSite($languages = null) 
+Installs Joomla with Multilingual Feature active
+
+ * `param`   array  $languages  Array containing the language names to be installed
+
+ * `example:` $I->installJoomlaMultilingualSite(['Spanish', 'French']);
+
+####  setErrorReportingToDevelopment() 
+Sets in Adminitrator->Global Configuration the Error reporting to Development
+
+@note: doAdminLogin() before
+
+####  installExtensionFromDirectory($path, $type = null) 
```

Comment:
mmm is automatically created. I will refactor the generator to avoid deprecated methods.